Postal.xcodeproj/project.pbxproj

```diff
@@ -786,7 +794,8 @@
 				PRODUCT_BUNDLE_IDENTIFIER = ai.snips.Postal;
 				SDKROOT = macosx;
 				SKIP_INSTALL = YES;
-				SWIFT_INCLUDE_PATHS = "$(SRCROOT)/dependencies $(SRCROOT)/dependencies/build/macos/include";
+				SWIFT_INCLUDE_PATHS = "$(inherited) $(SRCROOT)/dependencies/build/macos/include";
```

Comment:
I moved the `$(SRCROOT)/dependencies` path in the project settings because each target import it. And each target use it with `$(inherited)`.